### DISCUSSION TITLE
Keybindings with trigger are not shown with +list-keybinds

### DISCUSSION THREAD

**Discussion by hansemschnokeloch** [ghostty-org/ghostty#4065] [Feature Requests, Ideas]
Hi,

with this config file
```
keybind = clear
keybind = ctrl+f>l=new_split:right
keybind = ctrl+j=new_split:down
```
I get this output with +list-keybinds
```
14:40:18 ➜  ghostty +list-keybinds
ctrl  + j  new_split:down
```

Although the keybind with trigger is working fine

**Comment by rhodes-b** [comment]
To note this seems related to `keybind = clear`

without doing that the keybind lists for me
with it it does ont

**Comment by hansemschnokeloch** [comment]
With or without `keybind = clear` , I get the same output, and none of my keybindings with triggers are displayed
